test/acceptance/api_responses_test.go

```diff
@@ -31,7 +36,12 @@ func TestResponseForTransactionOnValidContract(t *testing.T) {
 }
 
 func TestResponseForTransactionOnContractNotDeployed(t *testing.T) {
-	NewHarness().Start(t, func(t testing.TB, parent context.Context, network *Network) {
+	NewHarness().WithSetup(func(ctx context.Context, network *Network) {
```

Comment:
should not stay ... will add new managememt memory provider and set it in the harness ctor.